src/foam/blob/Blob.js

```diff
@@ -154,6 +159,28 @@ foam.CLASS({
 
 foam.CLASS({
   package: 'foam.blob',
+  name: 'AbstractBlobService',
+  abstract: true,
+
+  implements: ['foam.blob.BlobService'],
+
+  methods: [
+    function put(blob) {
+      return this.put_(this.__context__, blob);
```

Comment:
Add an .in(x) like we do for DAO to return a new ProxyDAO to this in the specified context.

Reply:
PRoxyBlobService I mean

Reply:
@adamvy Done.